modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -12,57 +12,46 @@ For important extra information about your upgrade, see the release notes at htt
 ====
 
 
-.Procedure: Migrating the {productname} Server
-
 The [command]``server-migrator.sh`` script migrates {productname} Server to the latest version.
 It also upgrades the underlying operating system to version {opensuse-version}.
 The script is part of the [package]``susemanager`` package.
 
+
+The version of PostgreSQL you are using depends on the underlying operating system.
+{opensuse}{nbsp}Leap{nbsp}15.2 uses PostgreSQL{nbsp}12, so this is the version used by {uyuni} on that operating system.
+{opensuse}{nbsp}Leap{nbsp}15.3 is expected to have PostgreSQL{nbsp}13.
+
+
+
+.Procedure: Migrating the {productname} Server
 . Before running the [command]``server-migrator.sh`` script, check whether the most recent version of the [package]``susemanager`` package is installed:
 +
 ----
 zypper ref
 zypper up susemanager
 ----
-
 . Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
-// During migration the script will stop the spacewalk services.
```

Comment:
This comment was breaking the procedure:

![image](https://user-images.githubusercontent.com/3914967/111254360-99306480-8660-11eb-8e9d-42c12a64eb41.png)